code_challenges/code_wars/zip_with.py

```diff
@@ -0,0 +1,24 @@
+from operator import add, sub, mul
```

Comment:
unused imports should be removed.

Reply:
i have that there for my __if__ name == "__main__" block.

Reply:
Are all three used in your if name == main block?  If not, remove the ones that are not used.  Unused imports should not be present.